server/sandbox_run.go

```diff
@@ -254,7 +254,7 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 
 	securityContext := req.GetConfig().GetLinux().GetSecurityContext()
 	if securityContext == nil {
-		return nil, fmt.Errorf("no security context found")
+		securityContext = &pb.LinuxSandboxSecurityContext{}
```

Comment:
so, securityContext is mandatory and my chat with you was to keep changes like this just to your local working branch. We don't need them here if this is mandatory. My suggestion was to keep you going and avoid panic but again if this is mandatory no way the changes here are correct

Reply:
Confirmed with feisky that we should honor the kube config standard which is to always have something there.  fixing.